### PR TITLE
test: add unittest for pascal notation calculator

### PR DESCRIPTION
**Pull request by ksr1122**
Add test cases for basic, complex and divide by zero

### CODE REVIEW

File: test/calculator_test.cc [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+ public:
+  void SetUp() override {}
+  void TearDown() override {}
+  dev::Calculator calculator_;
+};
+
+// 1. Basic operations
+
+TEST_F(CalculatorTest, CalcAdd) {
+  ASSERT_EQ(3.0, calculator_.calc("1 + 2"));
```
Comment (LuxoftAKutsan) [inline test/calculator_test.cc]: https://github.com/google/googletest/blob/master/googlemock/docs/cheat_sheet.md#floating-point-matchers-fpmatchers

File: test/calculator_test.cc [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+TEST_F(CalculatorTest, CalcSub) {
+  ASSERT_EQ(-1.0, calculator_.calc("1 - 2"));
```
Comment (LuxoftAKutsan) [inline test/calculator_test.cc]: We are expecting calculator in polish notation https://en.wikipedia.org/wiki/Polish_notation

File: test/calculator_test.cc [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+TEST_F(CalculatorTest, CalcComplex2) {
+  ASSERT_EQ(-4.8, calculator_.calc("(1.5 - 2.5) * 4.8"));
+}
+
+// 4. Divide by zero operation
+
+TEST_F(CalculatorTest, CalcDivByZero) {
+  EXPECT_ANY_THROW(calculator_.calc("1.5 / 0"));
```
Comment (LuxoftAKutsan) [inline test/calculator_test.cc]: interesting topic about error handling

File: test/calculator_test.cc (hunk 1 above)
Comment (ksr1122) [reply test/calculator_test.cc]: I have used `ASSERT_DOUBLE_EQ` instead.

File: test/calculator_test.cc (hunk 2 above)
Comment (ksr1122) [reply test/calculator_test.cc]: updated.
